lib/src/wip/autocomplete/popup_controller.dart

```diff
@@ -23,6 +23,10 @@ class PopupController extends ChangeNotifier {
 
   int get selectedIndex => _selectedIndex;
 
+  void reset() {
```

Comment:
Add doc comment of why this is needed.
And a link to the bug in https://github.com/google/flutter.widgets